types/lru-cache/tslint.json

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json"
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false,
+        "member-access": false
```

Comment:
I made these rules more strict so I can write more strict type definitions

`no-unnecessary-generics` - this is necessary for the change in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59162/files#r820245041 That change improves DX because it allows the developer to hint about what type of data they should get back because it could be a more narrow type than what was defined in the constructor of the class.

`member-access` - TS assumes `public` access by default, and that's an "OK" default assumption, but there is benefit in explicitly declaring what is part of the public API.

Reply:
I'm not sure I understand why `public` would be more valuable in these types than in the others relying on the defaults.

In general, please suppress at line-level, rather than in the suppression file.

Reply:
Writing `public` is optional, but writing it makes the declarations more explicit in showing the author's intent about what is the public API should be ("Is this meant to be in the public API, or it's the default because that's how it is in the JS source code?").

I reverted changes to `tslint.json`, and now suppress false-positives in the relevant line/file.